packages/web-shared/ui-kit/ResourceCard/ResourceCard.js

```diff
@@ -89,10 +72,7 @@ function ResourceCard({
           <Styled.Title>
             <Styled.Ellipse
               width={{
-                _: '100px',
-                sm: '250px',
-                md: '350px',
-                lg: '500px',
+                _: '90%',
```

Comment:
Maybe it's only a problem if there is only one `responsive` style and it should be `"90%"` in that case? `maxWidth` change up above may need that tweak also.

Reply:
This caught my eye because I was trying to figure out why the action list item in your description picture was forcing the caret to the right when it had a long title.

Reply:
@nlewis84 Were you able to recreate that carret being forced to the left? I think I fixed that but accidentally  in dragged in an old screenshot but I'll check again :) 

that's a good callout regarding the `[object Object]`. I think the code is working, in that the styles are being applied, but it's broken that the `prop` makes it all the way down to the dom. It should def be redirected into the stylesheet.

Reply:
I could not recreate that issue 😄 
Not a blocking issue. This does seem to be working.

Reply:
Cool! I just confirmed right now also. You're right that I don't need the responsive object and I'm cleaning that up now before we merge